subprojects/docs/src/docs/userguide/running-builds/performance.adoc

```diff
@@ -191,6 +191,40 @@ Some collections have dedicated types, `TaskContainer` being one of them, that h
 If you do decide to use static compilation, using an IDE can quickly show errors due to unrecognised types, properties, and methods.
 You’ll also get auto-completion, which is always handy.
 
+=== Enable the configuration cache
+
+[IMPORTANT]
+====
+This feature is currently *_incubating_* and not enabled by default. The configuration cache does not support all core Gradle plugins and features. IDE imports and syncs do not use the configuration cache.
+====
+
+You can cache the result of the configuration phase by enabling the <<configuration_cache.adoc#config_cache,Configuration Cache>>. When build configuration input remains the same across builds, the configuration cache allows Gradle to skip the configuration phase entirely.
```

Comment:
... build configuration input**s** remain the same... ?

Reply:
Hm, maybe I'm not expressing this correctly. I'll sync up with you on Slack to see where the miscommunication is happening.

Reply:
I just meant that instead of saying _When build configuration input remains the same_, I'd use plural form  _inputs_ as in _When build configuration inputs remain the same_ like you do below.

Reply:
```suggestion
You can cache the result of the configuration phase by enabling the <<configuration_cache.adoc#config_cache,Configuration Cache>>. When build configuration inputs remain the same across builds, the configuration cache allows Gradle to skip the configuration phase entirely.
```